package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vanilla-framework",
-  "version": "3.12.1",
+  "version": "3.12.2",
```

Comment:
Since we are adding new "feature" (tagged logo) and new CSS class names, this should be 3.13.0.

Sorry, I know that versioning is quite unclear and arbitrary 😉

Reply:
Right, sorry about that. Set it to 3.13.0 now.